### PR TITLE
export parser.term.js

### PR DESCRIPTION
**Pull request by Nexucis**
It helps to have these terms exported when we have to analyze the tree for the linter. 
It will also help to have something similar of what is coding in prometheus: https://github.com/prometheus/prometheus/blob/fac7a4a0504404fa5d4c5abb8fcc9750bd5cbda7/promql/parser/parse.go#L424

### CODE REVIEW

**Comment by Nexucis** [comment]
it will also improve the autocompletion by doing this:

```typescript
import { AggregateExpr } from 'lezer-promql';

if (tree.type.id === AggregateExpr) {
}
```
instead of doing this:

```typescript
if (tree.name === "AggregateExpr") {
}
```

File: dist/index.d.ts [hunk 1]
```diff
@@ -2,3 +2,46 @@ import {Parser} from "lezer"
 
 export const parser: Parser
 export const promQLSyntax: { [key: string]: string[] }
+export const
+    LineComment: number,
+    Expr : number,
```
Comment (juliusv) [inline dist/index.d.ts]: The spacing around the colon is inconsistent, could you make them all `foo: bar` vs. `foo : bar`?

Comment (Nexucis) [reply dist/index.d.ts]: done

**Comment by Nexucis** [comment]
If you can wait before merging this PR. I'm currently translating this code: https://github.com/prometheus/prometheus/blob/master/promql/parser/parse.go#L433 and I'm not sure anymore it will be super helpful to have these constants exported ...

**Comment by juliusv** [comment]
Ok, will wait...

**Comment by Nexucis** [comment]
Should be fine at the end. Sorry for the delay and the mess

I'm still not able to iterate over the tree, but it will be a nice improvement to have these constants exported. It will reduce the number of magic string used.

**Comment by Nexucis** [comment]
and I think we should do like you said in a previous PR. Removing the map exported from the file `promql.term.js` and instead exporting multiple constants, one per list. Like that:

```typescript
export const matchOpTerms = [
        "=",
        "!=",
        "=~",
        "!~"
];
export const binOpTerms: [
        "^",
        "*",
        "/",
        "%",
        "+",
        "-",
        "==",
        ">=",
        ">",
        "<",
        "<=",
        "!=",
        "and",
        "or",
        "unless"
];
```

If you are ok with that, should I open another PR to make the changes ? or it is ok if I'm pushing the changes here ?

**Comment by juliusv** [comment]
Fine to do it here, I think!

**Comment by Nexucis** [comment]
thanks :). Should be fine now !

**Comment by juliusv** [comment]
:+1:

**Pull request merged**